.scalafix.conf

```diff
@@ -0,0 +1,35 @@
+rules = [
+  RemoveUnused
+  ExplicitResultTypes
+  Disable
+]
+
+ExplicitResultTypes {
+  unsafeShortenNames = true
+}
+
+Disable.ifSynthetic = [
+  "java/io/Serializable"
+  "scala/Any"
+  ## "scala/Product" # triggers even on non-synthetic
```

Comment:
This file should only contain rules we want to have and almost sure work well for us. Please remove the commented parts and maybe put some comments why something is enabled or a link to some explanation.

Reply:
this is the comment why it's disabled
```
# triggers even on non-synthetic
```
the file is originally from [`scalaz/scalazzi`](https://github.com/scalaz/scalazzi/blob/master/scalafix.conf)
but I wouldn't suggest fullblown scalazzi now, this is just a preparation, so that we can make the step to enforce in the future